src/frontend/js/widgets/Dashboard/components/DashboardAvatar/index.tsx

```diff
@@ -1,6 +1,32 @@
-import { User } from 'types/User';
+import c from 'classnames';
 
-export const DashboardAvatar = ({ user }: { user: User }) => {
-  const letter = user.username.charAt(0).toUpperCase();
-  return <div className="dashboard__avatar">{letter}</div>;
+export enum DashboardAvatarVariantEnum {
+  DEFAULT = 'default',
+  SQUARE = 'square',
+}
+
+interface DashboardAvatarProps {
+  title: string;
+  imageUrl?: string;
+  variant?: DashboardAvatarVariantEnum;
+}
+
+export const DashboardAvatar = ({
+  title,
+  imageUrl,
+  variant = DashboardAvatarVariantEnum.DEFAULT,
+}: DashboardAvatarProps) => {
+  const letter = title.charAt(0).toUpperCase();
+  return (
+    <div
+      data-testid="dashboard-avatar"
+      className={c('dashboard-sidebar__container__header__avatar', {
+        dashboard__avatar__square: variant === DashboardAvatarVariantEnum.SQUARE,
```

Comment:
From my point of view `square` acts like a modifier so we should use the following syntax : `dashboard__avatar--square`